src/drv_oculus_rift/packet.c

```diff
@@ -9,9 +9,11 @@
 
 
 #include <stdio.h>
+#include <string.h>
```

Comment:
this include seems unused, remove if possible please.

Reply:
Needed for memcpy